pom.xml

```diff
@@ -29,7 +29,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.509</version>
+    <version>1.609.1</version>
```

Comment:
It is easier to test coördinated changes across several plugins when they are all using the same baseline, which must be 1.609.x for https://github.com/jenkinsci/workflow-plugin/pull/147. If you insist, I can try to backdate the core requirement, but does it really matter? This plugin has not had a release since October, and the downstream plugins are all considered more or less experimental. IMO you should not be using them at all if you are on a conservative core version.